homeworks/hw1/hw1_template.py

```diff
@@ -1,58 +1,158 @@
 import argparse
+import os
+import sys
 
-import numpy as np
 import pandas as pd
 import torch
 import torch.nn as nn
 import torch.optim as optim
-from sklearn.metrics import accuracy_score, confusion_matrix
+from sklearn.metrics import accuracy_score, classification_report, confusion_matrix
 
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../../")))
```

Comment:
Looks like hardcode. Try using the '-m' option when running the script from the project root folder

Reply:
I fixed it. Running using python -m homeworks.hw1.hw1_template